sdk/tables/azure-data-tables/azure/data/tables/_models.py

```diff
@@ -459,6 +471,7 @@ def service_properties_deserialize(generated: GenTableServiceProperties) -> Dict
 
 
 class TableItem(object):
+    # pylint: disable=docstring-missing-param
```

Comment:
Why disable it rather than adding the docstring?

Reply:
This is weird. The docstring for the only pram `name` is there in class level and constructor, but still got `docstring-missing-param` error.

Reply:
Try use param instead of ivar.